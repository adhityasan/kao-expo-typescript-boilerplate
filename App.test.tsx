import React from "react";
import { create } from "react-test-renderer";

import App from "./App";

describe("<App />", () => {
  it("renders correctly", () => {
    const tree = create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("has 1 child", () => {
    const tree: any = create(<App />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});
