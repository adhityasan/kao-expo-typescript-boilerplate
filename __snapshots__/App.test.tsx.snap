// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View>
    <Text>
      Kao Expo Typescript Template
    </Text>
    <Text>
      Open up App.tsx to start working on your app!
    </Text>
  </View>
</View>
`;
